client/src/components/App/App.jsx

```diff
@@ -9,11 +8,36 @@ import PlaylistDetail from '../PlaylistDetail/PlaylistDetail'
 import useToken from '../../utils/useToken'
 import RecommendView from '../RecommendView/RecommendView'
 import UserPlaylistDetail from '../UserPlaylistDetail/UserPlaylistDetail'
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 
 function App() {
-  const [signupMessage, setSignupMessage] = useState('')
   const {token, setToken, clearToken} = useToken()
 
+  const notifySuccess = (successMessage) => {
```

Comment:
These notifications could be in their own file. One thing I recommend is keeping your App.jsx as light as possible. This way your app is a 'composition' of libraries..

Reply:
Oops, no you are right, I forgot that putting them in one file was an option...